api/v1alpha5/virtualmachinesnapshot_types.go

```diff
@@ -73,11 +73,45 @@ type QuiesceSpec struct {
 const (
 	// VirtualMachineSnapshotReadyCondition represents the condition
 	// that the virtual machine snapshot is ready.
+	// It's ready only when both VirtualMachineSnapshotCreatedCondition
+	// and VirtualMachineSnapshotCSIVolumeSyncedCondition are true.
 	VirtualMachineSnapshotReadyCondition = "VirtualMachineSnapshotReady"
 
-	// VirtualMachineSnapshotInProgressReason represents that a
-	// snapshot is in progress.
-	VirtualMachineSnapshotInProgressReason = "VirtualMachineSnapshotInProgress"
+	// VirtualMachineSnapshotWaitingForCreationReason documents that the virtual machine
+	// snapshot is waiting for creation.
+	VirtualMachineSnapshotWaitingForCreationReason = "VirtualMachineSnapshotWaitingForCreation"
+
+	// VirtualMachineSnapshotWaitingForCSISyncReason documents that the virtual machine
+	// snapshot is waiting for CSI volume sync.
+	VirtualMachineSnapshotWaitingForCSISyncReason = "VirtualMachineSnapshotWaitingForCSISync"
```

Comment:
Can you please group these together so all the creation condition and reasons are together in a block.  Ditto for the CSI condition and reasons.